### PR TITLE
Issue #799 add a terminal object for tty/pts and other pseudo terminals

### PR DESCRIPTION
**Pull request by mikeradka**
#### Description of changes:
- This PR adds an attribute to the `session` object for logging the pseudo-terminal, a valuable piece of information for detection engineering. For more info see Issue #799.

image

### CODE REVIEW

File: objects/session.json [hunk 1]
```diff
@@ -22,6 +22,10 @@
       "description": "The identifier of the session issuer.",
       "requirement": "recommended"
     },
+    "terminal": {
+      "description": "The Pseudo Terminal assicoated with the session. Ex: the tty or pts value.",
```
Comment (floydtree) [inline objects/session.json:26]: nit - spelling error in associated

**Review by floydtree — CHANGES_REQUESTED** [review]

File: objects/session.json (hunk 1 above)
Comment (mikeradka) [reply objects/session.json]: Nice catch, thanks Rajas.

**Review by floydtree — APPROVED** [review]

**Review by Aniak5 — APPROVED** [review]
LGTM!

**Review by zschmerber — APPROVED** [review]
Looks good to me.

**Pull request merged**